src/angular/src/app/masthead-docs/masthead-docs.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sprk-masthead-docs',
+  template: `
+      <div class="drizzle-o-ContentGrouping">
+        <h2 class="drizzle-b-h2">Masthead</h2>
+        <sprk-masthead 
+          additionalClasses="angular-docs-u-Masthead-docs"
+          clientName="Rob Copeland"
```

Comment:
clientName is application specific - something like username might work better or greeting name?